pkg/common/common.go

```diff
@@ -39,12 +39,20 @@ func CommonRendering(data embed.FS, pathtorender string, i interface{}) (string,
 	return tplBuffer.String(), nil
 }
 
-func GetGolangByteArray(data []byte) string {
+func GetGolangByteArray(data []byte, lang string) string {
```

Comment:
Maybe change the name to GetLanguageByteArray ?

Reply:
I dont know if a module for function specific to the language might be better, I will think about it

Reply:
I think your first comment is the solution i preferred.  For example, both rust and golang would use golangs current branch of this function